### PR TITLE
I added cities in texas

### PR DESCRIPTION
**Pull request by abiolakekky147**

### CODE REVIEW

**Comment by abiolakekky147** [comment]
please review. thank you!

File: file1 [hunk 1]
```diff
@@ -27,4 +27,10 @@ NJ
 CT
 OH
 CHICAGO
-
+=============================================
```
Comment (etech-staff) [inline file1:30]: thanks you

**Review by etech-staff — APPROVED** [review]

**Comment by abiolakekky147** [comment]
thank you!

**Comment by etech-staff** [comment]
thank you for your great work

**Pull request merged**